Source/Particles/Collision/BinaryCollision/NuclearFusion/ProtonBoronFusionInitializeMomentum.H

```diff
@@ -104,7 +104,9 @@ namespace {
             soa_2.m_rdata[PIdx::uz][idx_2], m2,
             ux_alpha1, uy_alpha1, uz_alpha1, static_cast<amrex::ParticleReal>(m_alpha),
             ux_Be, uy_Be, uz_Be, static_cast<amrex::ParticleReal>(m_beryllium),
-            E_fusion, engine);
+            E_fusion,
+            true, // isotropic scattering
```

Comment:
I like the way @EZoni handles the keywords. He does
```suggestion
            /*isotropic scattering=*/true,
```

Maybe we should agree on a WarpX standard format at some point.

Reply:
Thanks, I agree!

Reply:
Thanks for pointing this out, @roelof-groenewald. 

My idea would be to use the exact input argument name in the comment, rather than a comment itself. 

So if I were to suggest a convention, I would suggest in this case
```
/*isotropic_scattering=*/true,
```
since `isotropic_scattering` (with the `_`) is the exact input argument name. 

I think this can make searching easier, for example if one searches `isotropic_scattering`. 

It also mimics the Python keyword syntax, `f(x=1.0, y=2.0)`, where `x` and `y` match the input argument names.